assets/javascripts/discourse-assign/initializers/extend-for-assigns.js.es6

```diff
@@ -592,6 +870,7 @@ export default {
       });
     }
 
+    withPluginApi("0.13.0", (api) => makeTopicChanges(api));
```

Comment:
I don't understand the name here, can we find something more descriptive please?